Dockerfile

```diff
@@ -18,6 +18,14 @@ ENV READONLY_CACHE=true
 ###################
 FROM builder-cache-${use_cache} as builder
 
+# Information for Sentry Releases.
```

Comment:
Is this the Dockerfile that builds the `ci-wpcom` image?

Reply:
Actually no, that one appears to be created in the `Dockerfile.base` file. The .base one is executed once every 24 hours on a schedule and re-creates base images including a cache that webpack and other things can use.

This docker file is run on every commit to produce the build used in e2e tests and which gets deployed to production.